docs/config.json

```diff
@@ -476,6 +468,14 @@
             }
           ]
         },
+        {
+          "title": "Device Trust",
```

Comment:
I placed it right below "Login Rules", which is another preview feature. Not sure if we have a set ordering for these, so let me know what you think.